da/da_test.go

```diff
@@ -274,12 +266,12 @@ func doTestSubmitLargeBlocksOverflow(t *testing.T, dalc *DAClient) {
 
 	// overflowing blocks submit partially
 	resp := dalc.SubmitBlocks(ctx, []*types.Block{block1, block2}, limit, -1)
-	assert.Equal(StatusSuccess, resp.Code, "overflowing blocks should submit partially")
+	assert.Equal(resp.Message, "", "overflowing blocks should submit partially")
```

Comment:
Why assert for `resp.Code` was removed? Can't we just add assertion regarding `resp.Message`?

Reply:
The error message is being swallowed by `resp.Message`. Will restore it for now.